### DISCUSSION TITLE
Request  sample GEDZIP (.gdz) files in Example Files on GEDCOM.io

### DISCUSSION THREAD

**Discussion by emyoulation** [FamilySearch/GEDCOM#313] [Ideas]
In the [Example FamilySearch GEDCOM 7.0 Files](https://gedcom.io/tools/#example-familysearch-gedcom-70-files) in the Tools section of the GEDCOM.io site, there are 2 files: 
* maximal70.ged 
* minimal70.ged

It would be useful to have GEDZIP expansions of both files: 
* maximal70.gdz 
* minimal70.gdz
_(the embedded Media file might be a GEDCOM or GEDZIP icon in SVG MIME type, or a text file describing the Media type guidelines)_

Further, it would be good to have a testing guideline document. The testing standard should not be onerous and create a barrier to entry. And the evils of an elitist 'approved and endorsed' pogrom should be avoided. 

It seems reasonable that a good test should be a multi-pass test... where there should be no appreciable variation or increase in size after the 3rd import->export cycle. (The first couple passes might add [crosswalk](https://en.wiktionary.org/wiki/crosswalk)ing notations that export as custom tags.  But the importer should be able to handle their own crosswalking custom tags in a way that does not creating recursions on subsequent exports.) _I am reminded of the concept of the Star Trek transporter [test cylinders](https://memory-alpha.fandom.com/wiki/Test_cylinder) that were calibrated measures of standard that had to be repeatedly converted to verify no data was going to be lost... which might kill someone. However, we don't need developers feeling as though they are being forced to create [Heisenberg Compensators](https://memory-alpha.fandom.com/wiki/Heisenberg_compensator)!_

It might be reasonable to invite some people with a history in GEDCOM quality assurance testing to help with a MINIMAL test plan. Such as [Tamura Jones](https://www.tamurajones.net/GEDCOMValidation.xhtml), Andrew Hoyle (GEDCOM Validator™), Tim Forsythe (VGED), or one of the other aftermarket tools... maybe even some who was involved with the original GedChk

**Comment by tychonievich** [comment]
I agree we need example gdz files; we have an open issue for that on the GEDOM.io repository (https://github.com/FamilySearch/GEDCOM.io/issues/77)

I have no experience with validation or testing of implementations, but would welcome participation from someone who does. I'd also welcome the creation of a separate testing working group because I (perhaps incorrectly) think that most of the work of making tests and designing testing processes is independent of the process of defining and clarifying the spec and that both deserve attention without being made to compete with one another.

**Reply by emyoulation** [reply]
The [open issue 77](https://github.com/FamilySearch/GEDCOM.io/issues/77) was completely bare.  Added a reply with the suggestion for what the most basic 2 samples should include.

I agree that the validation or testing is a special skillset and deserves a separate working group.

Tamura Jones has done yoeman's service for the GEDCOM standard.  Arguably, his Oct. 2019 proposal of a 5.5.5 GEDCOM lit a fire under LDS to re-establish their copyright in Nov 2019 by officially releasing the 5.5.1 Final that had been in draft limbo since 1999.  And he has been doing structured torture testing of how well Genealogy tools handled GEDCOM files since 2009 and made software publishers accountable by publishing his results. He deliberately has not released his test files or methodology to prevent designing for the tests.  But hopefully he would be in favor of a baseline test that developer COULD build toward passing... since some many can't even get out of the gate in his current testing.  (Such as having invalid Header information.)

**Comment by emyoulation** [comment]
Topic resolved in thread mentioned above. New sample files added.
